_truststores/01_msft_removal_solutions.md

```diff
@@ -5,17 +5,15 @@ title: Microsoft Removal Solutions
 collection: truststores
 permalink: truststores/microsoft/
 ---
-In FY 2019, the U.S. Government is removing the Federal Public Key Infrastructure (FPKI) root certificate from the Microsoft certificate store. This change will affect all federal agencies and may have an impact on the following services:  
+In FY 2019, the U.S. Government is removing the Federal Public Key Infrastructure (FPKI) root certificate from the Microsoft certificate Trust Store. This change will affect all federal agencies and may have an impact on the following services:  
```

Comment:
revert this change please.  we agreed upon certificate store for this announcement based on the common terms used in general microsoft technet searches.

Reply:
Will do.  Thanks